src/main/java/com/smartbear/postman/script/PostmanScriptParser.java

```diff
@@ -64,7 +64,8 @@ public void parse(LinkedList<Token> tokens, ScriptContext context) throws SoapUI
     private boolean isEndOfCommand() {
         return lookahead.getType() == TokenType.END_OF_COMMAND ||
                 lookahead.getType() == TokenType.OPEN_CURLY_BRACKET ||
-                lookahead.getType() == TokenType.CLOSE_CURLY_BRACKET;
+                lookahead.getType() == TokenType.CLOSE_CURLY_BRACKET ||
+                lookahead.getType() == TokenType.NEW_LINE;
```

Comment:
New line is not a correct symbol of the end of command. Each command of a multi-line script should end with a semicolon.

The problem in the bug is that "exec" array contains 3 single-line scripts that we combine into one big script during parsing. I think a more correct fix would be to add a semicolon to the end of scriptLine (if missing) here: https://github.com/SmartBear/readyapi-postman-plugin/blob/9bd9c050d950ea2ee8bd065f73103e4b8d402009/src/main/java/com/smartbear/postman/collection/PostmanCollection.java#L53.